src/impl_/pymethods.rs

```diff
@@ -284,22 +272,6 @@ where
     retval
 }
 
-pub(crate) struct PyMethodDefDestructor {
```

Comment:
I'm happy to get rid of this :)

Reply:
Me too! It felt so awkward before to have to do the dance with `&'static str` in a bunch of places. In some we created these destructors and in others we made an unsafe requirement of no interior nulls. This end state is much more satisfying.